web/src/hashchange.js

```diff
@@ -72,6 +72,28 @@ function is_somebody_else_profile_open() {
     );
 }
 
+function handle_invalid_users_section_url(user_settings_tab) {
+    const valid_user_settings_tab_values = new Set([
+        "active-users-admin",
+        "deactivated-users-admin",
+        "invites-list-admin",
```

Comment:
These are user-facing names, since they appear in URLs, and we should probably pick something that actually looks nice. I see no reason for these to have extra tokens beyond just `active/deactivated/invites`.

We should probably discuss in #design, including whether `user-list-admin` should be renamed; I think `#organization/users/active` would be a much cleaner URL scheme for this section of the panel.

Reply:
That sounds great as a scheme; I'd be fine with moving forward without a #design discussion.

Reply:
OK but probably we should call it `/invitations` to match the new title.

Reply:
I've renamed `user-list-admin` to `users` along with a redirect in ca22e48b8546ba89aa4f0f4545d2f32f6e8bc055, as a preparatory commit and changed the tab_values in the main commit.